src/main/java/nl/novi/techiteasycontroller/service/TelevisionService.java

```diff
@@ -0,0 +1,159 @@
+package nl.novi.techiteasycontroller.service;
+
+import nl.novi.techiteasycontroller.dtos.TelevisionInputDto;
+import nl.novi.techiteasycontroller.dtos.TelevisionOutputDto;
+import nl.novi.techiteasycontroller.models.Television;
+import nl.novi.techiteasycontroller.repositories.TelevisionRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class TelevisionService {
```

Comment:
Er zit veel herhaling in het "mappen" van de objecten, je zou hier ook een functie voor kunnen schrijven die dat doet zo hoef je het maar 1x te typen.